### PR TITLE
Move torch.export() to torch.export.export()

### PR DESCRIPTION
**Pull request by gmagogsfm**
New plan:

torch.export.export() as the main API

All other utilities will be torch.export.foo_utilities

### CODE REVIEW

**Comment by gmagogsfm** [comment]
@msaroufim There is concern about mixing export and compile APIs in torch.compiler namespace since they do not inter-op. So we are creating a new namespace called torch.export to hold all export stuff and the main API call is now torch.export.export()

**Comment by gmagogsfm** [comment]
@qihqi FYI, the main API call name will change in order to avoid confusion.

**Review by tugsbayasgalan — APPROVED** [review]

File: torch/export/__init__.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+import torch
+
+
+__all__ = [
+    "export",
+]
+
+
+def export(
```
Comment (msaroufim) [inline torch/export/__init__.py:11]: LGTM, just make sure this renders correctly on pytorch.org

**Review by msaroufim — APPROVED** [review]

**Pull request closed without merging**